website/src/content/docs/docs/release-notes/release-2025-04-22.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "1.0.0-rc.1"
+releaseDate: 2025-04-22
+version: "1.0.0-rc.1"
+---
+
+## Features
+
+### @typespec/compiler
+
+- [#7067](https://github.com/microsoft/typespec/pull/7067) Adding support for nested paging properties.
+- [#6862](https://github.com/microsoft/typespec/pull/6862) `--trace` cli option applies to all commands now
+- [#7065](https://github.com/microsoft/typespec/pull/7065) Adds a TypeKit for Tuple types
+- [#7049](https://github.com/microsoft/typespec/pull/7049) Adds a new createDiagnosable typekit helper for APIs that return diagnostics
+- [#7018](https://github.com/microsoft/typespec/pull/7018) Removes the default typekit in favor of always instantiating typekits with either a `program` or `realm`.
```

Comment:
Should move this to breaking change? (Sorry - I know it's not marked as such in chronus)